### ISSUE TITLE
Wrong 'title/url' for page event

### ISSUE THREAD

**Issue by kwolniak** [dansmaculotte/nuxt-segment#41]
I have default settings for `nuxt-segment` so it sends automaticaly page event on navigation (route change). But in segment debugger I see that it sets wrong title.

Somehow it sets `title` and `url` from previus route...

EDIT: I see it is related if we use head function to set title etc.

**Comment by campbellchri** [comment]
@kwolniak did you find a workaround for this?  I've been experiencing the same

**Comment by kwolniak** [comment]
> @kwolniak did you find a workaround for this? I've been experiencing the same

@campbellchri yes, I created my own nuxt plugin implementation and override basic page properties via segment middleware
https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/middleware/#using-source-middlewares
so, I no longer use this package
